onnxruntime/core/providers/dml/DmlExecutionProvider/src/FusedGraphKernel.cpp

```diff
@@ -45,14 +47,17 @@
                 ORT_THROW_IF_FAILED(providerExecutionObject.As(&m_winmlProvider));
             }
 
-            TranslateAndCompileGraph(kernelInfo, graph, node.InputDefs(), node.OutputDefs(), graphNodePropertyMap, transferredInitializerMap);
+            TranslateAndCompileGraph(kernelInfo, graph, node.InputDefs(), fusedNodeInputArgOriginalNames, 
+                node.OutputDefs(), fusedNodeOutputArgOriginalNames, graphNodePropertyMap, transferredInitializerMap);
```

Comment:
[nit] Consistency with code nearby (line 79). (and ragged wrap sucks anyway for readability - much less visually scannable 👀)

Reply:
Just confirming whether you mean this?
TranslateAndCompileGraph(
                kernelInfo, 
                graph, 
                node.InputDefs(), 
                fusedNodeInputArgOriginalNames, 
                node.OutputDefs(), 
                fusedNodeOutputArgOriginalNames, 
                graphNodePropertyMap, 
                transferredInitializerMap);

Reply:
Yep, great. 👍